pyiode/pyiode_model.pyx

```diff
@@ -6,45 +6,246 @@
 # IODE Model functions
 # --------------------
 # 
-#   model_simulate(sample_from: str, sample_to: str, eqs_list=None, endo_exo_list=None,
-#                    eps: float = 0.0001, relax: float = 1.0, maxit: int = 100, 
-#                    init_values: int = KV_INIT_TM1, sort_algo: int = SORT_BOTH, nb_passes: int = 5, 
-#                    debug: int = 0, newton_eps: float = 1e-6, newton_maxit: int = 50, newton_debug: int = 0
-#                  )    |   Simulate a model
-#   model_calc_scc 
-#   model_simulateparms 
-#   model_simulate_scc 
-#   model_exchange 
-#   model_compile 
-#   model_simulate_save_niters 
-#   model_simulate_save_norms
-
-from pyiode_model cimport IodeModelSimulate
-
-
-def model_simulate(sample_from: str, sample_to: str, eqs_list=None, endo_exo_list=None,
-                    eps: float = 0.0001, relax: float = 1.0, maxit: int = 100, 
-                    init_values: int = KV_INIT_TM1, sort_algo: int = SORT_BOTH, nb_passes: int = 5, 
-                    debug: int = 0, newton_eps: float = 1e-6, newton_maxit: int = 50, newton_debug: int = 0
-                   ):
+#   model_simulate(sample_from: str, sample_to: str, 
+#                   eqs_list=None, endo_exo_list=None,
+#                   eps: float = 0.0001, 
+#                   relax: float = 1.0, 
+#                   maxit: int = 100, 
+#                   init_values: int = KV_INIT_TM1, 
+#                   sort_algo: int = SORT_BOTH, 
+#                   nb_passes: int = 5, 
+#                   debug: int = 0, 
+#                   newton_eps: float = 1e-6, 
+#                   newton_maxit: int = 50, 
+#                   newton_debug: int = 0)          
+#  
+#   model_calc_scc(nb_passes: int = 5, 
```

Comment:
arguments with default values MUST be placed AFTER all arguments with no default values

Reply:
No, they have default values. The correct syntax is 👍 
```
model_calc_scc(nb_passes: int = 1, 
                   pre_listname: str = "_PRE", 
                   inter_listname: str = "_INTER", 
                   post_listname: str = "_POST", 
                   eqs_list = None)
```

Reply:
Mmm OK. So the comment here is wrong.